File: Part4/Rust/Function_IncorrectFunctionName.rs
use std::io;

fn main()
{
    let mut user_input = String::new();
    
    println!( "Type 1 and press enter." );
    io::stdin().read_line( &mut user_input ).expect( "Failed to read line" );
    
    // Function name is wrong - should be parse
    let entered_integer: i32 = user_input.trim().string_to_number::<i32>().unwrap();
    
    println!( "The user entered the integer {}", entered_integer );
}

File: Part4/C#/Function_IncorrectFunctionName.cs
using System;

class Program
{
    static void Main()
    {
        string userInput;
        
        Console.WriteLine("Type 1 and press enter.");
        userInput = Console.ReadLine();
        
        // Function name is wrong - should be int.Parse
        int enteredInteger = int.ParseInt(userInput);
        
        Console.WriteLine("The user entered the integer {0}", enteredInteger);
    }
}
